sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.3.0-beta.2 (Unreleased)
 
 ### Features Added
+- Integrate synchronous workflow for sync clients so that they do not block on async client APIs.
```

Comment:
This isn't a feature to callout in changelogs.

Reply:
Do you have suggestions for it?

Reply:
Do you suggest not mentioning it in the changelog?

Reply:
I think it would be a good feature to highlight that the synchronous API's do not hold reactor chaining anymore.
cc: @g2vinay what do you think?

Reply:
We can mention it under Other Changes.
Also add to the sentence Simplifies stack traces and improves debugging experience.